logrotate/src/LogRotateHelper.ipp

```diff
@@ -298,7 +298,7 @@ bool LogRotateHelper::rotateLog() {
         is.seekp(0, std::ios::end);
         int length = is.tellp();
         is.seekp(0, std::ios::beg);
-        if (length > max_log_size_) {
+        if (length >= max_log_size_) {
```

Comment:
IMHO a nitpicking bugfix. But I don't mind fixing it.

However, shouldn't the test first catch the issue (and then the test pass after the fix?)